django/template/defaulttags.py

```diff
@@ -94,7 +94,7 @@ def render(self, context):
         from pprint import pformat
         output = [force_text(pformat(val)) for val in context]
         output.append('\n\n')
-        output.append(pformat(sys.modules))
+        output.append(force_text(pformat(sys.modules)))
```

Comment:
pformat uses cStringIO.StringIO() in Python 2. It would be safer to encode sys.modules before wrap it by pformat().